File: C Programs/Factorial.c
int fact(int n);
main()
{
   int n;
   printf("Enter a Number: ");
   scanf("%d",&n);

   printf("factorial is %d",fact(n));

}

int fact(int n)
{
    int f=1;
    while(n)
    {
        f=f*n;
        n--;
    }
    return (f);
    /*int i,f=1;
    for(i=1;i<=n;i++)
        f=f*i;
    return f;
  */}

File: Java Programs/factorial.java
public class factorial
{
 public static void main(String [] args)
 {
	 int num = 5,fact=1,i;
	 for(i=1;i<=num;i++)
	 {
		 fact = fact * i;
	 }
	 System.out.println("Factorial of "+num+" is "+fact);
 
}
}